client/src/utils/csv_parser.ts

```diff
@@ -144,7 +144,16 @@ function isValidEmail(email: string): boolean {
 }
 
 function isValidPhoneNumber(phoneNumber: string): boolean {
-  return phoneNumber.length === 10;
+
+  // Check if phone number has +64 at the start, replace it with 0
+  if (phoneNumber.startsWith('+64')) {
+    phoneNumber = phoneNumber.replace('+64', '0');
```

Comment:
edge case where u have +64[11-digit number]. Then this will be 12, and then fail the check?